elements/drawtools/src/enums/draw.js

```diff
@@ -0,0 +1 @@
+export const TRANSITION_DURATION = { duration: 750 };
```

Comment:
The naming might be a bit confusing here, since these are actually the fit options (as described in the [OL docs](https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#fit), so maybe `TRANSITION_OPTIONS` or `FIT_OPTIONS` could be better?

Reply:
Yeah this make sense. I will change it to `FIT_OPTION`, that will go much similar to OL Docs.